### PR TITLE
Update the overview and configuration sections

### PR DESCRIPTION
**Pull request by skovic**
This PR incorporates changes based on our ID team's review of the document.

### CODE REVIEW

File: physical_infra_provider_configuration/configuration/topics/setting_up_lxca.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+=== Setting up Lenovo XClarity Administrator
+
+Physical-infrastructure providers are used to connect CloudForms to one or more Lenovo XClarity Administrator instances. Lenovo XClarity Administrator is a centralized, resource-management solution that simplifies infrastructure management, speeds responses, and enhances the availability of Lenovo® server systems and solutions. It runs as a virtual appliance that automates discovery, inventory, tracking, monitoring, and provisioning for server, network, and storage hardware in a secure environment.
```
Comment (rodneyhbrown7) [inline physical_infra_provider_configuration/configuration/topics/setting_up_lxca.adoc]: These two paragraphs do not seem to fit in this section. I think we should remove them altogether and just leave the last couple of sections that reference downloading LXCA and the Pro- license Note.

File: physical_infra_provider_configuration/configuration/topics/index.adoc [hunk 2]
```diff
@@ -1 +1,6 @@
-== Configuring the Physical Infrastructure Provider
\ No newline at end of file
+== Configuring Lenovo XClarity Administrator
+
+Lenovo XClarity Administrator virtual appliances are connected to CloudForms by adding physical-infrastructure providers. Physical-infrastructure providers enable you to manage and interact with devices that are managed by Lenovo XClarity Administrator. After the physical-infrastructure provider is added, the servers that are managed by the Lenovo XClarity Administrator are registered in CloudForms. Managed devices can be monitored, configured, and updated using Lenovo XClarity Administrator, and the physical-infrastructure provider can surface details for the associated Lenovo XClarity Administrator  instance using the CloudForms user interface.
```
Comment (rodneyhbrown7) [inline physical_infra_provider_configuration/configuration/topics/index.adoc]: The last sentence here is wordy. Can we simplify this?     Managed devices can be monitored, configured and updated in Lenovo XClarity Administrator.  The CloudForms provider surfaces information about these managed devices and the XClarity Administrator instance.

File: physical_infra_provider_configuration/overview/topics/index.adoc [hunk 3]
```diff
@@ -1 +1,16 @@
-== Overview
\ No newline at end of file
+== Overview
+
+Lenovo® Physical Infrastructure Provider provides IT administrators the ability to integrate the management features of Lenovo XClarity Administrator with the hybrid-cloud management capabilities of Red Hat CloudForms. Lenovo expands the physical-infrastructure management for on-premise cloud configurations by leveraging Lenovo hardware management. It provides the configuration, monitoring, event management, and power monitoring needed to reduce cost and complexity through server consolidation and simplified management.
```
Comment (rodneyhbrown7) [inline physical_infra_provider_configuration/overview/topics/index.adoc]: The Lenovo Physical Infrastructure Provider ....

**Comment by skovic** [comment]
@rodneyhbrown7 I believe I have addressed all your comments. Please let me know if I missed anything. Thanks

**Pull request merged**